I.write-tests/03-remove-vowels-from-array/remove-vowels-in-array.test.js

```diff
@@ -1,9 +1,13 @@
+var removeVowelsForWords = require('../03-remove-vowels-from-array/remove-vowels-in-array')
 test('remove vowels from all words in array', function () {
     // Arrange
-
+    const words = ["Irina", "Etza", "Daniel"];
```

Comment:
What would happen if we add `Ashley` to this array?
The expected output would be `shl`. Can we add this and check if our function still works?
And if it doesn't, we should fix it

Reply:
It's  still work